backend/apid/graphql/error.go

```diff
@@ -17,7 +18,7 @@ type stdErr struct {
 func newStdErr(input string, err error) stdErr {
 	out := stdErr{code: schema.ErrCodes.ERR_INTERNAL, input: input}
 	switch terr := err.(type) {
-	case actions.Error:
+	case client.APIError: // TODO APIError
```

Comment:
Not TODO after all with the changes to `mapServiceErrCode()` below? Or is it referring to something else?